storm-core/test/clj/org/apache/storm/supervisor_test.clj

```diff
@@ -299,6 +299,7 @@
                                 "-Dstorm.conf.file="
                                 "-Dstorm.options="
                                 (str "-Dstorm.log.dir=" file-path-separator "logs")
+                                (str "-Djava.io.tmpdir=/tmp/workers" file-path-separator mock-worker-id file-path-separator "tmp")
```

Comment:
Actually, this is not what is under test here, so I stand corrected. The code that would use this path is stubbed/mocked I think.